lib/between_meals/util.rb

```diff
@@ -69,18 +69,22 @@ def execute(command)
     end
 
     def port_open?(port)
-      begin
-        Timeout.timeout(1) do
-          begin
-            s = TCPSocket.new('localhost', port)
-            s.close
-            return true
-          rescue Errno::ECONNREFUSED, Errno::EHOSTUNREACH
-            return false
+      ips = Socket.ip_address_list
+      ips.map! { |ip| ip.ip_address }
+      ips.each do |ip|
+        begin
+          Timeout.timeout(1) do
+            begin
```

Comment:
This seems backwards to me. We'll return the port is open if we can open it on *any* IP address... but we dont' know what address TT will try to bind to, so we could cause a problem.

I would think we would want to ensure we can listen to that port on **all** addresses if we're gonna do anything.

Reply:
Nope, I'm an idiot, we're checking if something listening, and if something is listening on this port on _any_ address, we want to return `true` which causes this port to NOT be used by TT.